### PR TITLE
Yo252yo

### PR DESCRIPTION
**Pull request by yo252yo**
Push request for formerly discussed table sorting

### CODE REVIEW

**Comment by Rybadour** [comment]
Hey, just budding in because I had some time after I submitted my own PR. Looks good to me except for my fairly minor suggestion :)

**Comment by PedroFonseca** [comment]
> Hey, just budding in because I had some time after I submitted my own PR. Looks good to me except for my fairly minor suggestion :)

What was your suggestion? Couldn't see it. Thanks for the review tho :)

Also i made an improvement so that the columns show an icon indicating which column the table is sorted by. Also added the possibility of sortying asc and desc by clicking again on same column. And finally also added sorting to the stores table.
Would appretiate if you could check it out.
Thanks for your contribuition :)

**Comment by Rybadour** [comment]
Oh, I just realized you have to submit the review for it to appear!

Those icons look good! I see no further issues :)

File: src/pages/Market/ProductsTable.tsx [hunk 1]
```diff
@@ -7,21 +7,35 @@ import { Accessor, For } from "solid-js";
...
   setShowStoreModal: (storeName: string) => void;
+  setOrdering: (ordering: Orderings) => void;
 };
 export default (props: Props) => (
   <Table>
     <TableHeader>
-      <TableHeaderCol>Product Name</TableHeaderCol>
-      <TableHeaderCol>Store Name</TableHeaderCol>
+      <TableHeaderCol>
+        <a onClick={() => props.setOrdering(Orderings.PRODUCT)}>
```
[the remark this answers is not in the document]
Comment (PedroFonseca) [reply src/pages/Market/ProductsTable.tsx]: Done. Thanks :)

**Comment by yo252yo** [comment]
Thanks a lot for the extra code! It's pretty much what i was going to do if i had managed to find time earlier, so we're in agreement ^^ wouldnt have found the icons though xD

**Pull request merged**